### DISCUSSION TITLE
dropped support for Hifiberry AMP2?

### DISCUSSION THREAD

**Discussion by jeroenterheerdt** [rern/rAudio#1397] [Bug Reports]
The Hifiberry AMP2 was supported in [i20230521](https://github.com/rern/rAudio/releases/tag/i20230521), but doesn't seem to be supported in https://github.com/rern/rAudio/releases/tag/i20230620. I know i20230620 is a pre-release, so maybe that's why, but just wondering what's happening here. I had to go back to i20230521 because of this. (Note, DACberry AMP2 is not the same as Hifiberry AMP2)

**Comment by EarsofTin** [comment]
I have a Hifiberry AMP2 on top of a RPi 3B updated through 20230630. Everything running fine.  Note that I select Hifiberry DAC+ as my Audio I2S device. I don't recall AMP2  ever being in the menu and per Hifiberry the AMP2 dac is equivalent to the DAC+

**Comment by jeroenterheerdt** [comment]
thanks, @EarsofTin it was / is on the list for i20230521, but I will try selecting DAC+, thanks!
